doc/tutorial/ranges/range_solution3.cpp

```diff
@@ -0,0 +1,22 @@
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/range/view/all.hpp>
+#include <seqan3/std/ranges>
+
+int main(int argc, char** argv)
+{
+    if (argc == 1)
+        return 0;
+
+    std::string s{argv[1]};
+
+    auto s_as_dna = s | seqan3::view::char_to<seqan3::dna5>;
+    // Bonus:
+    //auto s_as_dna = s | std::view::transform([] (char const c)
+    //{
+    //    return seqan3::assign_char_strict(seqan3::dna5{}, c);
+    //});
+
+    seqan3::debug_stream << "Original: " << s_as_dna << '\n';
+    seqan3::debug_stream << "RevComp:  " << (s_as_dna | std::view::reverse | seqan3::view::complement) << '\n';
```

Comment:
You did not include view::complement or view::translate... :thinking: probably somewhere included in the debug_stream header. For the tutorial I would include the headers explicitly

Reply:
EDIT: ops, you include `range/all.hpp` :woman_facepalming:  can you add a comment behind the include? I actually thought you included the view::all of ranges